CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.12)
 project(DISTOPIA LANGUAGES C CXX)
 
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
👍

Reply:
This might be a problem for older compilers, what 17 features are we using?

Reply:
I was using it for `aligned_alloc`, which turns out we may not need. Some of the type traits stuff may require 17 but could probs get away with 14 or possibly even 11. Best to double check with @nbgl?